include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -232,7 +232,12 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
                    "_%06T"};
 
             plugins::multi::Option<std::string> jsonConfig
-                = {"json", "advanced (backend) configuration for openPMD in JSON format", "{}"};
+                = {"json", "advanced (backend) configuration for openPMD in JSON format (used when writing)", "{}"};
+
+            plugins::multi::Option<std::string> jsonRestartConfig
+                = {"jsonRestart",
```

Comment:
Please add the new parameter to the coresponding `rst` file and `TBG_macros.cfg`

Reply:
done